### DISCUSSION TITLE
Game crashes when giving contextual order on an item during cpr

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13908] [Bug reports]
### What happened?

Game crashes when giving contextual order (middle clicking) on an item in a patient's inventory while doing cpr on them.

https://github.com/FakeFishGames/Barotrauma/assets/37850646/0872152a-bad1-4608-a31f-25650eef7f59

Game crashed immediately upon middle clicking the screwdriver

### Reproduction steps

1. give cpr
2. middle click item in their inventory
3. Oops! Barotrauma has crashed

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

**Comment by HawkinsThe71st** [comment]
[crashreport (33).log](https://github.com/FakeFishGames/Barotrauma/files/15169000/crashreport.33.log)

**Comment by itchyOwl** [comment]
Thanks for the report, looking into it!

**Comment by Rokvach** [comment]
Issue has been fixed and tested. Working correctly, closing discussion.
